src/Overrides/TenancyUrlGenerator.php

```
@@ -105,41 +106,49 @@ class TenancyUrlGenerator extends UrlGenerator
      */
-    public function temporarySignedRoute($name, $expiration, $parameters = [], $absolute = true)
+    public function toRoute($route, $parameters, $absolute)
     {
-        if ($name instanceof BackedEnum && ! is_string($name = $name->value)) {
-            throw new InvalidArgumentException('Attribute [name] expects a string backed enum.');
-        }
+        $name = $route->getName();
 
-        $wrappedParameters = Arr::wrap($parameters);
+        if ($name) {
+            [$prefixedName, $parameters] = $this->prepareRouteInputs($name, Arr::wrap($parameters));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Arr' in method 'toRoute'. (undefined)

(StaticAccess)